crates/kitsune_p2p/proxy/src/proxy_url.rs

```diff
@@ -32,21 +32,21 @@ impl ProxyUrl {
     /// Create a new proxy url from a full url str.
     pub fn from_full(full: &str) -> TransportResult<Self> {
         macro_rules! err {
-            () => {
-                TransportError::from(format!("Invalid Proxy Url: {}", full))
+            ($h:literal) => {
+                TransportError::from(format!("Invalid Proxy Url({}): {}", $h, full))
             };
```

Comment:
```suggestion
            ($h:literal) => {
                TransportError::from(format!("Invalid Proxy Url({}): {}: at: {}:{}", $h, full, file!(), line!()))
            };
```
Could be super useful to include the  file and line number.